.editorconfig

```diff
@@ -14,6 +14,9 @@ indent_size = 4
 # code alone for now, we overwrite max_line_length for TypeScript code.
 max_line_length = 100
 
+[.github/**.yml]
```

Comment:
`ya?ml`, just in case?

Reply:
Figured out how to make that work, good idea :D (`y{,a}ml` it was)

Reply:
That's a weird pattern...